core/metrics_manager.js

```diff
@@ -94,387 +479,6 @@ MetricsManager.FixedEdges;
  *          }}
  */
 MetricsManager.UiMetrics;
```

Comment:
Regarding this `@typedef` (and the previous three similar ones):

* I'd have suggested making these separate named exports, only it looks like we don't have _any_ named exports from this module.
* Can they be unexported `const`s?
* If not then I'd suggest they be exported, possibly after being factored out into a separate module that is `require`d both by this one any by any client code that needs these types.
* Otherwise perhaps they could be declared as `static` properties in the `class MetricsManager`  declaration